whylog/config/abstract_config.py

```diff
@@ -1,18 +1,27 @@
+import itertools
 from abc import ABCMeta, abstractmethod
 from collections import defaultdict
 from datetime import datetime
 
 import six
 
+from whylog.config.filename_matchers import WildCardFilenameMatcher
 from whylog.config.investigation_plan import Clue, InvestigationPlan, InvestigationStep
+from whylog.config.log_type import LogType
 from whylog.config.parser_name_generator import ParserNameGenerator
 from whylog.config.parser_subset import ConcatenatedRegexParser
 from whylog.config.rule import RegexRuleFactory
+from whylog.config.super_parser import RegexSuperParser
 
 
 @six.add_metaclass(ABCMeta)
 class AbstractConfig(object):
     words_count_in_name = 4
```

Comment:
ok, but still it should be uppercase (unless you want to make a getter, but here you don't)